prow/jobs/jinja/presubmits/service_tests.jinja2

```diff
@@ -123,4 +123,25 @@
           value: {{ service }}
         command: ["make", "test"]
 
+  - name: {{ service }}-metadata-file-test
+    decorate: true
+    optional: false
+    always_run: true
+    annotations:
+      karpenter.sh/do-not-evict: "true"
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
```

Comment:
You need to include the `test-infra` repository as an extra ref, and set it as the default cwd, to use the new script

Reply:
Good catch. Added